File: src/main/java/uk/co/threebugs/mochiwhattotrade3/setupgroups/SetupGroups.java
package uk.co.threebugs.mochiwhattotrade3.setupgroups;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "setup_groups")
public class SetupGroups {

    private String name;

    private String scriptsDirectory;
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/setupgroups/SetupGroups.kt
package uk.co.threebugs.mochiwhattotrade3.setupgroups

import jakarta.persistence.*

@Entity
@Table(name = "setup_groups")
class SetupGroups (
    var name: String? = null,
    var scriptsDirectory: String? = null,
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int? = null
)
